src/content/en/tools/lighthouse/audits/budgets.md

```diff
@@ -0,0 +1,283 @@
+
+### Define a budget file {: #define }
+
+By convention the budget file is usually called `budget.json` but you can call it whatever you like.
+The example below sets the following budgets:
+
+* 125 kilobytes for all scripts
+* 50 kilobytes for all stylesheets
+* 100 network requests total
+
+```json
+[
+  {
+    "path": "/",
+    "resourceSizes": [
+      {
+        "resourceType": "script",
+        "budget": 125
+      },
+      {
+        "resourceType": "stylesheet",
+        "budget": 50
+      }
+    ],
+    "resourceCounts": [
+      {
+        "resourceType": "total",
+        "budget": 100
+      }
+    ]
+  }
+]
+```
+
+The table below describes each of the `budget.json` properties.
+
+<table>
+  <tbody>
+    <tr>
+      <th>Name</th>
+      <th>Type</th>
+      <th>Valid Value(s)</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <td><code>path</code></td>
+      <td>String</td>
+      <td>
+        A path pattern adhering to the
+        <a href="/search/reference/robots_txt#url-matching-based-on-path-values">robots.txt format</a>.
+      </td>
+      <td>
+        The <code>resourceSizes</code> and <code>resourceCounts</code> budgets apply to all
+        pages matching this pattern.
+      </td>
+    </tr>
+    <tr>
+      <td><code>resourceSizes</code></td>
+      <td>Array</td>
+      <td>
+        An array of objects. Each object must have a <code>resourceType</code> and <code>budget</code> property.
+      </td>
+      <td>
+        The <code>budget</code> value specifies the total size limit in kilobytes for the category that's specified
+        in <code>resourceType</code>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>resourceCounts</code></td>
+      <td>Array</td>
+      <td>
+        An array of objects. Each object must have a <code>resourceType</code> and <code>budget</code> property.
+      </td>
+      <td>
+        The <code>budget</code> value specifies the total resource count limit for the category that's specified
+        in <code>resourceType</code>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>resourceType</code></td>
+      <td>String</td>
+      <td>
+        <code>total</code>, <code>document</code>, <code>script</code>, <code>stylesheet</code>, <code>image</code>,
+        <code>media</code>, <code>font</code>, <code>other</code>, or <code>third-party</code>.
+      </td>
+      <td>
+        <code>total</code> measures all page resources. <code>document</code> measures HTML document requests.
+        <code>other</code> includes XHR or Fetch requests, and data transfers over WebSocket connections.
+        <code>third-party</code> measures all resources from third-party domains.
+      </td>
+    </tr>
+    <tr>
+      <td><code>budget</code></td>
+      <td>Integer</td>
+      <td>Any positive integer.</td>
+      <td>The file size limit or resource count limit.</td>
```

Comment:
"Depending on context, either the file size limit in kilobytes or the number of requests."

Reply:
Done